android/app/src/main/java/org/fossasia/openevent/activities/TrackSessionsActivity.java

```diff
@@ -134,6 +143,33 @@ private void loadData() {
 
             mSessions.clear();
             mSessions.addAll(track.getSessions().sort("startsAt"));
+
+//            finding upcoming and ongoing sessions
+            Date current;
+            int countUpcoming = 0;
+            int countOngoing = 0;
+            for (Session trackSession : mSessions) {
+                flag = 0;
+                Date start = new Date();
+                Date end = new Date();
```

Comment:
Um, does it make a difference if we do that. I'm not aware, could you tell that please?